<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetBaseWorker</class>
 <widget class="QWidget" name="WidgetBaseWorker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>746</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1,10">
   <item>
    <widget class="QGroupBox" name="groupBoxMessage">
     <property name="title">
      <string>员工基础信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="labelWorkerBarcode">
        <property name="text">
         <string>员工卡号：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelWorkerDuty">
        <property name="text">
         <string>员工职务：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDescription">
        <property name="text">
         <string>备注：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editWorkerBarcode"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelWorkerID">
        <property name="text">
         <string>员工编号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelWorkerName">
        <property name="text">
         <string>员工姓名：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBoxWorkerDuty"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editWorkerID"/>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QTextEdit" name="editDescription"/>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QLineEdit" name="editWorkerName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ButtonGroup" name="buttonGroup" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>员工列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonGroup</class>
   <extends>QWidget</extends>
   <header>ButtonGroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
